src/esp/sensor/CameraSensor.h

```diff
@@ -12,6 +12,19 @@
 namespace esp {
 namespace sensor {
 
+struct CameraSensorSpec : public VisualSensorSpec {
+  CameraSensorSpec() : VisualSensorSpec() {
+    uuid = "rgba_camera";
+    sensorSubType = SensorSubType::Pinhole;
+    parameters = {{"near", "0.01"},
```

Comment:
Like I mentioned in the 1:1 with you,
- Would you move the "near" "far" "hfov" etc. out of the parameters?
- Would you put them in the VisualSensor level temporarily, making them member variables? (I suspect we also need it in the fisheye sensor.)